app/components/DocsPopover.tsx

```diff
@@ -36,17 +36,21 @@ export const DocsPopoverLink = ({ href, linkText }: DocsPopoverLinkProps) => (
 )
 
 type DocsPopoverProps = {
-  heading: React.ReactNode
+  heading: string
   icon: JSX.Element
   links: Array<DocsPopoverLinkProps>
   summary: string
 }
 
 export const DocsPopover = ({ heading, icon, summary, links }: DocsPopoverProps) => {
+  const title = `Learn about ${heading}`
   return (
     <Popover>
-      <PopoverButton className={cn(buttonStyle({ size: 'sm', variant: 'ghost' }), 'w-8')}>
-        <Info16Icon aria-label="Links to docs" className="shrink-0" />
+      <PopoverButton
+        className={cn(buttonStyle({ size: 'sm', variant: 'ghost' }), 'w-8')}
+        title={title}
+      >
+        <Info16Icon aria-hidden className="shrink-0" />
```

Comment:
Confirmed this is sufficient for a11y and, indeed, likely preferable. There's no need to have the label on the icon.

<img width="497" alt="image" src="https://github.com/user-attachments/assets/44667539-743c-4c56-8f04-b9ed72471dad" />